Mathlib/RepresentationTheory/GroupCohomology/Basic.lean

```diff
@@ -184,20 +184,22 @@ noncomputable abbrev inhomogeneousCochains : CochainComplex (ModuleCat k) ℕ :=
       LinearEquiv.toModuleIso_inv, LinearEquiv.coe_coe, LinearEquiv.symm_apply_apply, this,
       LinearMap.zero_apply, map_zero, Pi.zero_apply] -/
     ext x
-    have := LinearMap.ext_iff.1 ((linearYonedaObjResolution A).d_comp_d n (n + 1) (n + 2))
-    simp only [ModuleCat.comp_def, LinearMap.comp_apply] at this
+    have : ∀ x, _ = (0 : _ →ₗ[_] _) x := LinearMap.ext_iff.1 (ModuleCat.hom_ext_iff.mp
+      ((linearYonedaObjResolution A).d_comp_d n (n + 1) (n + 2)))
+    simp only [ModuleCat.hom_comp, LinearMap.comp_apply] at this
     dsimp only
-    simp only [d_eq, LinearEquiv.toModuleIso_inv, LinearEquiv.toModuleIso_hom, ModuleCat.coe_comp,
-      Function.comp_apply]
+    simp only [d_eq, LinearEquiv.toModuleIso_inv_hom, LinearEquiv.toModuleIso_hom_hom,
+      ModuleCat.hom_comp, LinearMap.comp_apply, LinearEquiv.coe_coe, ModuleCat.hom_zero]
     /- Porting note: I can see I need to rewrite `LinearEquiv.coe_coe` twice to at
       least reduce the need for `symm_apply_apply` to be an `erw`. However, even `erw` refuses to
       rewrite the second `coe_coe`... -/
-    erw [LinearEquiv.symm_apply_apply, this]
-    exact map_zero _
+    erw [LinearEquiv.symm_apply_apply, this, LinearMap.zero_apply]
+    simp
+    rfl
```

Comment:
This proof got uglier, not sure what the concrete cause is.

Reply:
I tried a few things and it seems there was already a lot of defeq abuse that got exposed now.